deltacat/examples/common/fixtures.py

```diff
@@ -0,0 +1,54 @@
+import os
+import logging
+from deltacat import logs
+
+from deltacat.constants import (
+    DELTACAT_APP_LOG_LEVEL,
+    DELTACAT_SYS_LOG_LEVEL,
+    DELTACAT_APP_LOG_DIR,
+    DELTACAT_SYS_LOG_DIR,
+    DELTACAT_APP_INFO_LOG_BASE_FILE_NAME,
+    DELTACAT_SYS_INFO_LOG_BASE_FILE_NAME,
+    DELTACAT_APP_DEBUG_LOG_BASE_FILE_NAME,
+    DELTACAT_SYS_DEBUG_LOG_BASE_FILE_NAME,
+)
+
+logger = logs.configure_deltacat_logger(logging.getLogger(__name__))
+
+
+def create_runtime_environment():
+    # read packages to install in the runtime environment from system environment vars
+    package = os.environ.get("PACKAGE")
+    logger.debug(f"Pip Install Packages: {package}")
+
+    runtime_environment = None
+    if package:
+        # read the stage (e.g. alpha, beta, dev, etc.) from the system environment vars
+        stage = os.environ["STAGE"]
+        logger.debug(f"Job Run Stage: {stage}")
+
+        # log the job run system environment for debugging
+        logger.debug(f"Job Run System Environment: {os.environ}")
+
+        worker_env_vars = {
+            # forward the STAGE environment variable to workers
+            "STAGE": stage,
+            # forward deltacat logging environment variables to workers
+            "DELTACAT_APP_LOG_LEVEL": DELTACAT_APP_LOG_LEVEL,
+            "DELTACAT_SYS_LOG_LEVEL": DELTACAT_SYS_LOG_LEVEL,
+            "DELTACAT_APP_LOG_DIR": DELTACAT_APP_LOG_DIR,
+            "DELTACAT_SYS_LOG_DIR": DELTACAT_SYS_LOG_DIR,
+            "DELTACAT_APP_INFO_LOG_BASE_FILE_NAME": DELTACAT_APP_INFO_LOG_BASE_FILE_NAME,
+            "DELTACAT_SYS_INFO_LOG_BASE_FILE_NAME": DELTACAT_SYS_INFO_LOG_BASE_FILE_NAME,
+            "DELTACAT_APP_DEBUG_LOG_BASE_FILE_NAME": DELTACAT_APP_DEBUG_LOG_BASE_FILE_NAME,
+            "DELTACAT_SYS_DEBUG_LOG_BASE_FILE_NAME": DELTACAT_SYS_DEBUG_LOG_BASE_FILE_NAME,
+        }
+
+        # setup runtime environment from system environment variables:
+        #   1. STAGE: The execution environment stage (e.g. alpha, beta, dev, etc.)
+        #   2. PACKAGE: Package to Pip Install. (e.g. signed S3 URL for DeltaCAT)
+        runtime_environment = {
+            "env_vars": worker_env_vars,
+            "pip": package,
+        }
```

Comment:
Minor: Can these be part of glue job itself? Do we need to create a separate runtime env within the job?

Reply:
We can now get rid of `pip` (see conversations above), but don't know of any good way to get rid of the environment variables, which are currently used to configure our loggers appropriately during job run execution. @jjacobj84 @chappidim - please correct me if there's a better way to pass environment variables out to all workers in the cluster.

Reply:
AFAIK any non-reserved job parameters become env vars. @jjacobj84 @chappidim Please confirm.

Reply:
They do become environment variables, but these environment variables only appear to be propagated to the head node by default (you need to manually propagate them to worker nodes like this, unless there's a separate argument to set or this behavior is considered a bug on the AWS Glue side).

Reply:
Got it, seems like a bug. Don't see any reason for this to be intentional.